src/odb/include/odb/db.h

```diff
@@ -7555,7 +7581,11 @@ class dbModBTerm : public dbObject
   void disconnect();
   void staSetPort(void* p);
   void* staPort();
-
+  bool isBusPort() const;
+  void setBusPort(dbBusPort*);
+  dbBusPort* getBusPort() const;
+  dbModBTerm* getNext() const;
+  dbModBTerm* getPrev() const;
```

Comment:
Should be private

Reply:
unresolved

Reply:
These are used by the various iterators over modbterms. So they need a public interface. Or do you want the iterators to be made a friend of this low level class ? Or do you want the code duplicated so there is equivalent functionallity between the public and private members ? I think I am missing a coding pattern here.

Reply:
Can getNext/Prev now be removed as you have the new iterators?

Reply:
yes, done. also removed the backward traversal (no longer needed now reverse works)